### ISSUE TITLE
Alpha dashboard it not centered when open neo-tree

### ISSUE THREAD

**Issue by thieung** [goolord/alpha-nvim#110]
With dashboard-nvim, when I open neo-tree, it still centered. But after movibng to alpha-nvim, it's not. See below screenshot:
<img width="1705" alt="image" src="https://user-images.githubusercontent.com/79964592/167235322-fa102327-47ad-4d03-90b3-828778283ab9.png">

Should we make it always be centered like dashboard-nvim?

**Comment by goolord** [comment]
yes the intent is for it to be centered, alpha will redraw (and thus center) for nvim-tree, I didn't consider that another plugin like this wouldn't fire the same autocmd events..

**Comment by hermitmaster** [comment]
I have the same problem with nvim-tree.lua when setting the option `open_on_setup = true`.

**Comment by mrbjarksen** [comment]
I've experienced a similar issue with command line auto-completion (using nvim-cmp), where alpha is redrawn every time a new popup appears. This normally wouldn't be an issue, but I'm using a randomized header function so this is jarring. I assume both of these issues are caused by a WinNew autocommand. Perhaps this should be user configurable?

**Comment by latipun7** [comment]
It works on nvim-tree, but in neo-tree, when neo-tree is closed, the cursor is not in alpha menu, when I move cursor down to the menu, it works and alpha is centered.

https://user-images.githubusercontent.com/20012970/191427003-fc0a1593-a821-4cca-8a51-420d07550fcf.mp4

any workaround? thanks

**Comment by goolord** [comment]
note: this https://github.com/goolord/alpha-nvim/commit/1dcd325e2e5dcfe40de48013bc587adc5f74582a only works on neovim version 0.10 or greater, so if you're on an older version of neovim this will not work
